pyamg/dev/elasticity_bar.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
```

Comment:
I don't have a particular opinion on whether to keep them.  I just figured that you wanted to keep them around, which is totally fine (or not if you want to remove them).  But if we do keep them around, pyamg-examples is probably the best place.